### ISSUE TITLE
ACR Task run fails through extension

### ISSUE THREAD

**Issue by ehotinger** [Microsoft/vscode-docker#679]
1. Right click, run task file on a valid task file.
2. Choose a valid subscription and registry
3. Error

```
Setting up temp file with 'sourceArchive249914.tar.gz'
   Sending source code to temp file
   Getting build source upload URL 
   Getting blob info from upload URL 
   Creating blob service 
   Creating block blob 
Uploaded source code to /sourceArchive249914.tar.gz
Set up run request
Error: The provided source location 'source/201812050000/107924a9-1268-4ea7-b4b8-4c9ca7da7e5b.tar.gz' is invalid or doesn't exists. Please use 'GetBuildSourceUploadUrl' to get a valid location to upload the source.
```

**Comment by ehotinger** [comment]
Maybe this is fixed by https://github.com/Microsoft/vscode-docker/pull/656 - build has identical before. Not sure.

**Comment by StephenWeatherford** [comment]
@rosanch Rodrigo, can you verify this is fixed by #656?  Thx.

**Comment by rosanch** [comment]
I still have to test it on Linux. However, I did add changes to fix this in #656
The problem was in how the path was being handled

**Comment by rosanch** [comment]
This PR will be close because is the same issue in #666. Please follow that issue for more info.
